pyat/at/physics/magnet_tools.py

```diff
@@ -0,0 +1,162 @@
+"""Magnet tools."""
+
+from __future__ import annotations
+
+from typing import Sequence
+
+import numpy
+from scipy.special import comb
+
+__all__ = [
+    "feeddown_polynomba",
+    "feeddown_from_nth_order",
+    "feeddown_pol_from_element",
+]
+
+
+def feeddown_pol_from_element(ele,verbose: bool=False):
```

Comment:
Dear @swhite2401 ,
I have put the skeleton of a new function that takes an element and returns the polynoms. Is this what you were asking ?
If so, I will implement the steps in the commented lines.